src/main/java/org/baggle/domain/meeting/domain/Meeting.java

```diff
@@ -67,4 +60,13 @@ public void updateMeetingStatusIntoOngoing() {
     public void updateMeetingStatusIntoTermination() {
         this.meetingStatus = MeetingStatus.TERMINATION;
     }
+
+    public void updateParticipationList(Participation newParticipation) {
+        this.participations.forEach(participation -> participation.updateButtonAuthority(ButtonAuthority.NON_OWNER));
+        this.participations.add(newParticipation);
+    }
+
+    public void updateButtonAuthorityOfParticipationList(int randomNumber) {
+        this.getParticipations().get(randomNumber).updateButtonAuthority(ButtonAuthority.OWNER);
+    }
```

Comment:
만약 위에 제시된 메서드명으로 변경하게 된다면, 메서드명의 의미가 겹칠 것 같습니다. 따라서 아래와 같이 해당 메서드를 수정하고 
```
Participation randomNumberParticipation = meeting.getRandomNumberParticipation(randomNumber);
randomNumberParticipation.updateButtonAuthority(ButtonAuthority.OWNER);
```
와 같이 분리해 보는 것은 어떨까요?
```suggestion
    public Participation getRandomNumberParticipation(int randomNumber) {
        return getParticipations().get(randomNumber);
    }
```

Reply:
좋은 방법인것 같습니다!!